src/domains/nodo-common/02_security.tf

```diff
@@ -0,0 +1,54 @@
+resource "azurerm_resource_group" "sec_rg" {
+  name     = "${local.product}-${var.domain}-sec-rg"
+  location = var.location
+
+  tags = var.tags
+}
+
+module "key_vault" {
+  source = "git::https://github.com/pagopa/azurerm.git//key_vault?ref=v2.13.1"
+
+  name                       = "${local.product}-${var.domain}-kv"
+  location                   = azurerm_resource_group.sec_rg.location
+  resource_group_name        = azurerm_resource_group.sec_rg.name
+  tenant_id                  = data.azurerm_client_config.current.tenant_id
+  soft_delete_retention_days = 90
+  lock_enable                = true
+
+  # Logs
+  sec_log_analytics_workspace_id = var.env_short == "p" ? data.terraform_remote_state.core.outputs.sec_workspace_id : null
+  sec_storage_id                 = var.env_short == "p" ? data.terraform_remote_state.core.outputs.sec_storage_id : null
```

Comment:
```suggestion
```
in next days we'll manage diagnostic settings using azure policy so we can remove this